test/doc/traits/callable_constant.cpp

```diff
@@ -0,0 +1,35 @@
+#include <iostream>
+#include <type_traits>
+#include <eve/traits/overload.hpp>
+#include <eve/wide.hpp>
+
+namespace eve
+{
+  struct pi_t : constant<pi_t>
+  {
+    template<eve::floating_value T> auto call(as<T>) const -> T;
+
+    // A constant must provide a value() static member with two parameters:
+    //  * an eve::options pack containing potential decorators passed to the constant
+    //  * an eve::as instance to specify the output type
+    template<typename T> static constexpr auto value(auto, eve::as<T>) { return T{3.14159216}; }
+
+    EVE_CALLABLE_OBJECT(pi_t, pi_);
+  } inline constexpr Pi;
+};
```

Comment:
I like thism except for capital Pi.

How about `pi_example`

Reply:
Will change